lib/src/widgets/build_message_container.dart

```diff
@@ -119,29 +118,16 @@ Container buildMsgBox(
     }
   }
 
-  // screen dimensions
-
+  // Determine device type for layout adjustments
   final isMobile = size.width <= 730;
   final isTablet = size.width > 730 && size.width <= 1050;
 
-  var horizontalPadding = 0.0;
-  var leftSpace = size.width * 0.12;
-  final rightSpace = size.width * 0.12;
-
-  if (isMobile) {
-    horizontalPadding = size.width * 0.01;
-  } else if (isTablet) {
```

Comment:
It would be interesting to know how these fine-grained adjustments are derived and why they are no longer necessary.